tests/util/checker/README.md

```diff
@@ -57,16 +57,14 @@ type RulesFactory interface {
 
 ### Whitelist
 
-In addition to [RulesFactory](rule.go), application can provide a [Whitelist](whitelist.go) to opt out selected files
```

Comment:
If whitelisting is done using a comment, it should be documented here.

And Whitelist.go is no longer a first class citizen and there no need to mention it here anymore.